app/src/pages/metamine/visualizationNU/ScatterPlot.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="main tool_page wrapper-box">
+    <div class="md-layout md-alignment-top-left md-gutter adjust-padding" style="margin:10px">
+      <div class="main-content" style="margin: 1rem; display: flex">
+        <div class="scatter-chart" style="width: 50%">
+          <Scatter />
+        </div>
+        <div class="subcharts" style="width: 20%">
+          <Structure />
+          <Youngs />
+          <Poisson />
+        </div>
+        <div class="side-tools" style="width: 30%;">
+          <DataSelector />
+          <RangeSelector />
+          <MaterialInformation />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+.subcharts {
```

Comment:
I notice you have this `.subcharts` class scoped in a couple different components. Is the styling significantly different per component? If not, it might be preferable to move it to our assets/css directory